packages/design-carousel/src/styles.scss

```diff
@@ -23,19 +23,19 @@ $patterns: "17-2", "black", "ella-d", "link-cloud", "matt-smith", "ose-maiko", "
 .design-carousel__item {
 	flex: 1;
 
-	@include break-small {
+	@include break-medium {
 		aspect-ratio: 1.5;
 	}
 
 	&.design-carousel__item-mobile {
-		@include break-small {
-			display: none !important;
+		@include break-medium {
+			display: none;
 		}
 	}
 	&.design-carousel__item-desktop {
-		display: none !important;
-		@include break-small {
-			display: flex !important;
+		display: none;
+		@include break-medium {
+			display: flex;
```

Comment:
Small note: we may want to specify border-box here to ensure that the chrome around the screenshot appears in-screen and the screenshot doesn't span the whole screen. I have two screen captures to show the difference.
```suggestion
			display: flex;
			box-sizing: border-box;
```

#### Emulated iPad Air with code from PR

<img width="495" alt="Screenshot 2022-12-15 at 12 54 58" src="https://user-images.githubusercontent.com/3376401/207842007-46453905-f9f0-4c4d-a00d-b2dee98fb35f.png">

#### Emulated iPad Air with additional `box-sizing` declaration

<img width="495" alt="Screenshot 2022-12-15 at 12 55 45" src="https://user-images.githubusercontent.com/3376401/207842048-2985b64f-433d-4462-a14a-3860adcfad4d.png">